packages/ui/src/components/textarea.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+import { cn } from '../utils/cn'
+
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
```

Comment:
```suggestion
export interface TextareaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {}
```

Let’s import these types directly from react

```tsx
import { TextareaHTMLAttributes } from 'react'
```